my-app/src/components/card/card.component.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./card.css";
+import Name from "../name/name.component";
+import Avatar from "../avatar/avatar.component";
+import Role from "../role/role.component";
+import CardSection from "../cardSection/card-section.component";
+
+const Card = ({ name, mentor, mentee, section }) => {
+  return (
+    <div
+      className={
+        mentor && mentee
+          ? "mentor-mentee card"
+          : mentee
+          ? "mentee card"
+          : "card"
+      }
+    >
+      <div className={section.length > 1 ? "extended content" : "content"}>
+        <Avatar />
+        <Name name={name} />
+        <Role mentor={mentor} mentee={mentee} />
+        {section.length > 1 ? (
```

Comment:
fmi, why not >0? then the section.map code can be re-used and the ternary won't be necessary - it becomes [conditional rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator)